cmd/flightctl/main.go

```diff
@@ -209,21 +216,23 @@ func RunGet(kind, name string, labelSelector string, output string) error {
 			if err != nil {
 				return fmt.Errorf("reading device/%s: %v", name, err)
 			}
-			if response.HTTPResponse.StatusCode != http.StatusOK {
-				return fmt.Errorf("reading device/%s: %s", name, response.HTTPResponse.Status)
-			}
-
-			marshalled, err := yaml.Marshal(response.JSON200)
+			err = printSingleResourceResponse(response, fmt.Sprintf("device/%s", name))
 			if err != nil {
-				return fmt.Errorf("marshalling device: %v", err)
+				return err
 			}
-			fmt.Println(string(marshalled))
 		} else {
-			params := &api.ListDevicesParams{}
+			params := api.ListDevicesParams{}
 			if len(labelSelector) > 0 {
 				params.LabelSelector = &labelSelector
 			}
-			response, err := c.ListDevicesWithResponse(context.Background(), params)
+			if limit > 0 {
```

Comment:
you can make them pointers and avoid checking the value

Reply:
Done